src/core/core.tooltip.js

```diff
@@ -701,10 +701,10 @@ module.exports = function(Chart) {
 						// Inner square
 						ctx.fillStyle = mergeOpacity(vm.labelColors[i].backgroundColor, opacity);
 						ctx.fillRect(pt.x + 1, pt.y + 1, bodyFontSize - 2, bodyFontSize - 2);
-						var textColor = mergeOpacity(vm.labelTextColors[i], opacity);
-						ctx.fillStyle = textColor;
 					}
 
+					var textColor = mergeOpacity(vm.labelTextColors[i], opacity);
+					ctx.fillStyle = textColor;
 					fillLineOfText(line);
```

Comment:
You could combine these two lines into one:

    ctx.fillStyle = mergeOpacity(vm.labelTextColors[i], opacity);

It might be better for minification / download size that way